README.md

```diff
@@ -224,7 +224,30 @@ The OpenTelemetry Operator *might* work on versions outside of the given range,
 | v0.29.0                | v1.19 to v1.21       |
 | v0.28.0                | v1.19 to v1.21       |
 | v0.27.0                | v1.19 to v1.21       |
-| v0.26.0                | v1.18 to v1.20       |
+
+
+### OpenTelemetry Operator vs. Cert Manager
+Since Cert-Manager deprecated *cert-manager.io/v1alpha2, cert-manager.io/v1alpha3, cert-manager.io/v1beta1, acme.cert-manager.io/v1alpha2, acme.cert-manager.io/v1alpha3, acme.cert-manager.io/v1beta1* APIs on version 1.4.0, we decided to create a compability Matrix between OpenTelemetry Operator and Cert-manager.
```

Comment:
Perhaps we could have something like this in the previous section as well? Like: "We use `cert-manager` for some features of this operator and the third column shows the versions of the `cert-manager` that are known to work with this operator's versions."